regexes/client/browsers.yml

```diff
@@ -1334,6 +1353,9 @@
 - regex: 'UC[ ]?Browser.* \(UCMini\)'
   name: 'UC Browser Mini'
   version: ''
+- regex: 'Chrome.+uc mini browser(\d+[\.\d]+)?'
+  name: 'UC Browser Mini'
```

Comment:
Is android and chrome version > 28 + Safari/537.36 then BrowserEngine blink name

```
engine:
    versions:
      28: 'Blink'
```